logs_http.module

```diff
@@ -152,6 +146,27 @@ function logs_http_register_event(array $log_entry) {
   // Remove empty values, to prevent errors in the indexing of the JSON.
   $event = logs_http_array_remove_empty($event);
 
+  if (variable_get('logs_http_debugging_mode', FALSE)) {
+    if (!isset($events['debugging'])) {
+      $debugging_event = $event;
+      unset($debugging_event['type']);
```

Comment:
No, the debug Info should not be part of the event because each request can produce any number of events and we don't want to include the debug information into each of them as it will be the same content every single time. That's not useful.

We only want to create one extra event with the debugging information and that should be the first one in the series of events. And that'sexactly what the PR achieves.